pkg/providers/azure/README.md

```diff
@@ -170,13 +186,13 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     usepodidentity: "true"
     ```
 
-7. Deploy your app
+1. Deploy your app
 
     ```bash
     kubectl create -f examples/nginx-pod-secrets-store-inline-volume-pod-identity.yaml
     ```
 
-8. Validate the pod has access to the secret from key vault:
+1. Validate the pod has access to the secret from key vault:
 
     ```bash
     kubectl exec -it nginx-secrets-store-inline-pod-identity cat /kvmnt/testsecret
```

Comment:
nit: `cat /kvmnt/testsecret` should be changed to `cat /mnt/secrets-store/secret1`